packages/svelte-parse/src/void_els.ts

```diff
@@ -0,0 +1,25 @@
+export const void_els = [
```

Comment:
Can easily be a RegExp -- or a dictionary

```js
export const void_els = {
  area: 1,
  base: 1,
  // ..
}

// super fast; O(1)
void_els[tagName] !== void 0
```

Dictionary is actually better since there's not that too many.. no major size penalty